packages/postgres-database/src/simcore_postgres_database/utils_models.py

```diff
@@ -10,7 +11,13 @@ class FromRowMixin:
     """Mixin to allow instance construction from aiopg.sa.result.RowProxy"""
 
     @classmethod
-    def from_row(cls: type[ModelType], row: RowProxy) -> ModelType:
+    def from_row_proxy(cls: type[ModelType], row: RowProxy) -> ModelType:
         assert is_dataclass(cls)  # nosec
         field_names = [f.name for f in fields(cls)]
         return cls(**{k: v for k, v in row.items() if k in field_names})  # type: ignore[return-value]
+
+    @classmethod
+    def from_orm(cls: type[ModelType], row: Row) -> ModelType:
+        assert is_dataclass(cls)  # nosec
+        field_names = [f.name for f in fields(cls)]
+        return cls(**{k: v for k, v in row._asdict().items() if k in field_names})  # type: ignore[return-value]
```

Comment:
at some point we should deprecate this, as this has flows when in use with special types.
That is exactly why I did not like that we started this library without pydantic and without all our aliases.
But that is for another PR...